src/OrleansPSUtils/OrleansPSUtils.2017.csproj

```diff
@@ -0,0 +1,59 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup Label="NuGet">
+    <IsPackable>true</IsPackable>
+    <PackageId>Microsoft.Orleans.OrleansPSUtils</PackageId>
```

Comment:
I'll move the powershell module to the official nuget package: [Microsoft.PowerShell.5.ReferenceAssemblies/](https://www.nuget.org/packages/Microsoft.PowerShell.5.ReferenceAssemblies)